### PR TITLE
stdlib logging can handle printing and writing to files

### PR DESCRIPTION
**Pull request by l0rb**
don't reinvent the wheel. everything that can be handled by the `logging` module should be handled by it. reduces complexity of loggo  (and line numbers too)

### CODE REVIEW

**Comment by interrogator** [comment]
I think this should probably be merged, provided:

1. Filehandler builds intermediate directories before writing
2. Both handlers add tracebacks
3. Nobody needs the `loggo.write_to_file`, method, which was public so that the user could override it by subclassing (i.e. to print to different files for different services)

@l0rb can you confirm first two points? @hukkinj1 can best answer number 3

**Comment by l0rb** [comment]
1. is a no. FileHandler does not. Is that a requirement though?

**Comment by interrogator** [comment]
I don't know if requirement, but is possible with current code and not with proposed code, so obv it merits discussion. yes i think best handled by accepting this pr plus makedirs in __init__ or similar

**Comment by l0rb** [comment]
I'm open to changes if there is anything the code owners object to in this PR. Also note that the coverage % is only going down because this removes more lines than it adds, but all affected lines have full coverage.

**Comment by l0rb** [comment]
The latest commits fixed both issues. If the directory for the logs file does not exist it is created, and logs going to file or stdout have the traceback appended in the exact same style as they were before. Unfortunately flake disagrees with the stdlib.

File: loggo/loggo.py [hunk 1]
```diff
@@ -67,6 +68,17 @@ class Formatters(TypedDict, total=False):
 CallableEvent = Literal["called", "errored", "returned", "returned_none"]
 
 
+class _Formatter(logging.Formatter):
+    def __init__(self) -> None:
+        super().__init__("%(asctime)s\t%(message)s\t%(levelno)s", "%Y-%m-%d %H:%M:%S %Z")
+
+    def format(self, record: logging.LogRecord) -> str:
```
Comment (hukkinj1) [inline loggo/loggo.py]: We need `  # noqa: A003` at the end here.

File: loggo/loggo.py [hunk 2]
```diff
@@ -67,6 +68,17 @@ class Formatters(TypedDict, total=False):
...
+        msg = super().format(record)
+        if hasattr(record, "traceback"):
+            msg += " -- see below:\n" + record.traceback
```
Comment (hukkinj1) [inline loggo/loggo.py]: Need either a `  # type: ignore` here or rewrite as
```python
traceback = getattr(record, "traceback", None)
if traceback:
    msg += " -- see below:\n" + traceback
```

**Comment by hukkinj1** [comment]
Need to fix linting and merge conflicts before we can merge. I marked the lines where linting fails above ^

**Comment by hukkinj1** [comment]
Fixed conflicts and merged. Thanks @l0rb !

**Pull request merged**